docs/vo_conesearch/validator.rst

```diff
@@ -92,7 +92,7 @@ Each Cone Search service is a
 `~astroquery.vo_conesearch.vos_catalog.VOSDatabase`
 (see :ref:`vo-sec-client-cat-manip` and :ref:`vo-sec-client-db-manip`).
 
-In the master registry, there are duplicate catalog titles with
+In the main registry, there are duplicate catalog titles with
```

Comment:
would a renaming happen in VO land, too? Could we just call it the default rather than master/main?

```suggestion
In the default registry, there are duplicate catalog titles with
```